apiclient/harvester_api/managers.py

```diff
@@ -612,10 +612,24 @@ class VirtualMachineManager(BaseManager):
 
     Spec = VMSpec
 
+    def download_virtctl(self, *, raw=False, **kwargs):
+        code, info = self._get(f"apis/subresources.{self.VM_API}/v1/version")
+        version, platform = info['gitVersion'], info['platform']
+        resp = self.api.session.get("https://github.com/kubevirt/kubevirt/releases/download/"
+                                    f"{version}/virtctl-{version}-{platform}", **kwargs)
+        if raw:
+            return resp
+        else:
+            return resp.status_code, resp.content
+
```

Comment:
Maybe later we can have a more alternative url friendly version of acquiring `virtctl` - like if we had a file-server offline that already had it, so the location for the release wouldn't be tied directly to `github.com/kubevirt/releases/download/...` but something like `http://111.222.333.444:8080/virtctl-1.0.0-amd64`?
Maybe that URL path could just be abstracted to something in the `config.yml` if we need to later.

Reply:
The `virtctl` binary should always use the version reflect to the cluster, to prevent any incompatible functions.
(It is the reason the URL is fixed)

In airgaped test case, we can add an configure variable to download the `virtctl` file.  But I think the output of serial console should be the same between two environment, so we can just test those cases in general environment.